examples/base_time_add/sidecar_controller.py

```diff
@@ -0,0 +1,37 @@
+from osparc_control import ControlInterface
+
+
+control_interface = ControlInterface(
```

Comment:
maybe instead of *control_interface* you could call it *controlled_solver* ?

Reply:
I've renamed this to `PairedTransmitter`, which is the best thing that came to mind. I'm still open for discussion and possible name changes.